### PR TITLE
pkg/uefi/file: marshal file size to JSON

### PR DESCRIPTION
**Pull request by orangecms**

### CODE REVIEW

File: pkg/uefi/file.go [hunk 1]
```diff
@@ -173,13 +174,34 @@ const (
...
+		t[1] = b[1]
+	}
+	if len(b) > 2 {
+		t[2] = b[2]
+	}
+	return nil
+}
+
+func (t *ThreeUint8) MarshalJSON() ([]byte, error) {
+	res := uint32(t[0]) + uint32(t[1])<<8 + uint32(t[2])<<16
```
Comment (GanShun) [inline pkg/uefi/file.go:194]: Can you use Read3Size? There's already a helper function that does this.

File: pkg/uefi/file.go [hunk 2]
```diff
@@ -173,13 +174,34 @@ const (
 	FileStateValid FileState = FileStateHeaderConstruction | FileStateHeaderValid | FileStateDataValid
 )
 
+type ThreeUint8 [3]uint8
+
+func (t *ThreeUint8) UnmarshalJSON(b []byte) error {
+	if len(b) == 0 {
```
Comment (GanShun) [inline pkg/uefi/file.go:180]: replace with 
if copy(t[:], b) == 0 {
    return fmt.Errorf(...)
}
return nil

Comment (orangecms) [reply pkg/uefi/file.go]: done

File: pkg/uefi/file.go (hunk 1 above)
Comment (orangecms) [reply pkg/uefi/file.go]: now that you showed me how, yea, thank you :)

**Review by GanShun — APPROVED** [review]

**Pull request merged**